lib/modules/versioning/distro.ts

```diff
@@ -145,9 +167,16 @@ export class DistroInfo {
    */
   public getNLatest(n: number): DistroInfoRecordWithVersion | null {
     const len = this._sortedInfo.length - 1;
-    const i = len - Math.floor(n);
+    let i = len - Math.floor(n);
+    let j = 0;
+
+    while (!this.isReleased(this._sortedInfo[len - j].version)) {
```

Comment:
Please use a for loop, It needs to break before we get our of bounds, otherwise this will throw a null pointer for version